### DISCUSSION TITLE
LoCon training?

### DISCUSSION THREAD

**Discussion by cerarslan** [kohya-ss/sd-scripts#262] [General]
Hello sir, firstly, I want to say I appreciate for all yor effort. Idk I u saw that there is new a network algorithm which is LoCon. So maybe we can use in colab other thx to @Linaqruf   put link: 

https://github.com/KohakuBlueleaf/LoCon

**Comment by Linaqruf** [comment]
You can easily integrating LoCon with kohya script as module extension, just `pip install locon==0.0.2` , then set `network_module` to `locon.locon_kohya` and add additional args in `network_args` to `["conv_dim=(desired_number)","conv_alpha=(desired_number)"]` and you're good to go.

**Reply by cerarslan** [reply]
> You can easily integrating LoCon with kohya script as module extension, just `pip install locon==0.0.2` , then set `network_module` to `locon.locon_kohya` and add additional args in `network_args` to `["conv_dim=(desired_number)","conv_alpha=(desired_number)"]` and you're good to go.

Well.. I lost after pip install locon==0.0.2 part :'). I only be able to use your colab 1-click kohya trainer and I'm close to my colab limit :'/

**Reply by cerarslan** [reply]
Oh I just saw the kohya support LoCon but I don't know how to train and test on your colab mate :(
